app/bundles/LeadBundle/Form/Type/FieldType.php

```diff
@@ -275,21 +275,19 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                     $cleaningRules['defaultValue'] = 'raw';
 
                     if (is_array($data)) {
-                        $properties = $data['properties'] ?? [];
+                        $properties = $data['properties_template_select_template'] ?? [];
```

Comment:
Now I see that changing this won't help. I need to figure it out how the lookup field for can also have `$data['properties']`.

Reply:
I spend some time figuring out how and why the lookup field doesn't have the proper structure. Maybe that's the reason why, when creating this type of field, it breaks.

Any inputs from anyone will be really helpful.